cloud/kubernetes/bring-your-own-certs/client.yaml

```diff
@@ -1,4 +1,4 @@
-# Generated file, DO NOT EDIT. Source: cloud/kubernetes/templates/bring-your-own-certs/client.yaml
+# Generated file, DO NOT EDIT. Source: cockroachdb_cockroach_master/cloud/kubernetes/templates/bring-your-own-certs/client.yaml
```

Comment:
Are these changes expected/desirable? If yes, LGTM

Reply:
We're good to merge this PR once checks pass 👍🏼 :shipit: 

This is expected:
- Previously this was generated from a dev's local machine/repository.
- Now this is being generated from a TC job (sandbox environment where this repository is installed in order to generate this PR)

Reply:
I think we can improve this by removing the prefix.

Reply:
👍🏼 tracking in https://cockroachlabs.atlassian.net/browse/RE-476